demos/embeddings/barbell-graphwave.ipynb

```diff
@@ -0,0 +1,162 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "m1 = 7\n",
+    "m2 = 7\n",
+    "gnx = nx.barbell_graph(m1=m1, m2=m2)\n",
+    "nx.draw(gnx)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### GraphWave embedding calculation\n",
+    "\n",
+    "Now, we're ready to calculate the GraphWave embeddings. We need to specify the points to sample the characteristic function. We need to specify: \n",
+    "\n",
+    "- the wavelet `scales` to the use. This is a graph dependant hyperparameter. Experiment with different values or use the `scales=\"auto\"` option\n",
+    "- the number of eigen vectors to use `num_eigenvecs`. A larger number of eigenvectors will yield better results but is more computationally expensive. `num_eigenvecs=None` will calculate all eigenvectors.\n",
+    "- the `sample_points` at which to sample the characteristic function. This should be: `sample_points=np.linspace(0, max_val, number_of_samples)` and is graph dependant."
```

Comment:
```suggestion
    "- the `sample_points` at which to sample the characteristic function. This should be of the form `sample_points=np.linspace(0, max_val, number_of_samples)` and is graph dependent."
```

or 
```suggestion
    "- the `sample_points` at which to sample the characteristic function. This should be of the form `sample_points=np.linspace(0, max_val, number_of_samples)`. The best value depends on the graph."
```
("dependant" is a person who depends on someone else, i.e. something that appears on tax forms, etc.)